src/jailer/src/main.rs

```diff
@@ -328,6 +328,33 @@ fn sanitize_process() {
     }
 }
 
+#[cfg(target_arch = "aarch64")]
+// Safe because this syscall takes static parameters and
+// we check the output.
+unsafe fn enable_ssbd_mitigation() {
```

Comment:
I'd rather have the function safe, and only the `libc::prctl` wrapped in an `unsafe` block

Reply:
Yes, the `unsafe fn` is a remnant of an earlier implementation and I forgot to remove it, will do.

Reply:
Removed.